ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolScheduleBuilder.java

```diff
@@ -184,15 +185,21 @@ protected Stream<Optional<BuilderMapEntry>> createMilestones(
   }
 
   @Override
-  protected void postBuildStep(final MainnetProtocolSpecFactory specFactory) {
+  protected void postBuildStep(
+      final MainnetProtocolSpecFactory specFactory, final TreeMap<Long, BuilderMapEntry> builders) {
```

Comment:
This approach will be neater when we unify the schedules because we can do away with postBuildStep